lib/build-view.js

```diff
@@ -158,10 +162,16 @@ export default class BuildView extends View {
     if (0 === w || 0 === h) {
       return;
     }
-
+    const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
+    if ( orientation === 'Left' || orientation === 'Right' ) {
+      this.cache[orientation].width = $('.build .output').width();
+    } else if ( orientation === 'Top' || orientation === 'Bottom' ) {
+      this.cache[orientation].height = $('.build .output').height();
```

Comment:
I'm not too sure what the best approach is here. This did fix the issue I described but agree that there should be a more complete solution. I'll take a look at it again.